src/integration-test/java/org/ow2/proactive/catalog/rest/controller/CatalogObjectControllerIntegrationTest.java

```diff
@@ -121,7 +121,7 @@ public void cleanup() {
     @Test
     public void testCreateWorkflowShouldReturnSavedWorkflow() {
         given().pathParam("bucketName", bucket.getName())
-               .queryParam("kind", "workflow")
+               .queryParam("kind", "Workflow/specific-workflow-kind")
```

Comment:
the Workflow with first letter uppercase is correct ?

Reply:
yes, It's required to test case-insensitive 
So the user can put any capital letters